AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/EditorEntityComponentProperty.py

```diff
@@ -0,0 +1,18 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+This util is intended to store common test values for editor component tests.
+"""
+
+from editor_python_test_tools.editor_entity_utils import EditorComponent
+
+
+class EditorEntityComponentProperty:
+
+    def __init__(self, editor_component: EditorComponent, path: str, clamped: bool = True):
+        self.property_path = path
+        self.editor_component = editor_component
+        self.clamped = clamped
```

Comment:
Remove Clamped.

This is unused and is a forward vision for implementation. 

Having this here early, and defaulting to true, is not providing any value to the libraries at this time, especially when we haven't determined the pattern for how to handle clamping (EG: you'd have to pass in some sort of data object with the clamped bounds when true to be stored and managed on the child class).

Reply:
removed the clamped parameter.